addons/web_tip/static/src/js/tip.js

```diff
@@ -260,10 +258,23 @@
             }
             return def;
         },
+ 
+        //if needed, scroll prior to displaying the tip
+        scroll_tip: function(){
+            var self = this;
+            var scroll = _.find(self.$element.parentsUntil('body'), function(el) {
+                var overflow = $(el).css('overflow-y');
+                return (overflow === 'auto' || overflow === 'scroll');
+            });
+            if (scroll) {
+                $(scroll).scrollTo(self.$element);
+            }
+        },
 
         end_tip: function(tip) {
             var self = this;
             var Tips = new instance.web.Model('web.tip');
+            $('#' + self.$element.attr('aria-describedby')).remove();
```

Comment:
Why is it this needed ? Doesn't the `popover('destroy')` call handle the removal from the dom ?